crates/modules/logger/src/lib.rs

```diff
@@ -65,11 +66,33 @@ async fn logger_task(
     }
 }
 
+#[derive(Clone, Debug)]
+enum OutputFormat {
+    Plaintext,
+    Json,
+}
+
+impl FromStr for OutputFormat {
+    type Err = ConfigError;
+    fn from_str(format: &str) -> Result<Self, Self::Err> {
+        match format {
```

Comment:
add support for uppercase option

```suggestion
        match format.to_lowercase().as_str() {
```

Reply:
Done